server/controller/FlightController.js

```diff
@@ -515,17 +517,17 @@ router.put(
 async function runChecksStartCalculationsStoreFixes(
   flightDbObject,
   userId,
+  // TODO: Why like this?
```

Comment:
What do you mean exactly?

Reply:
Was es mit dem {foo: false} = {} auf sich hat. Hab ich nicht gerafft.

Reply:
https://www.mixmax.com/engineering/default-options-in-es6